src/recoilState/merkleDistributor.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  MerkleDistributorInfo,
+  parseBalanceMap,
+} from 'helpers/merkle-distributor/parse-balance-map';
+import { selectorFamily } from 'recoil';
+
+import { rReceivedGiftsByUser } from './receivedGiftsByUser';
+
+import { IRecoilGetParams } from 'types';
+
+export const rMerkleDistributor = selectorFamily<
```

Comment:
maybe `rEpochRoot` is a better name for this

Reply:
`rEpochMerkleRoot`